### PR TITLE
Filter by metadata

### PR DESCRIPTION
**Pull request by krzysztofwolski**

### CODE REVIEW

File: docs/developer/metadata.mdx [hunk 1]
```diff
@@ -140,3 +140,43 @@ mutation {
   }
 }
 ```
+
+
+## Filter by metadata
+Objects with metadata interface can be filtered by it's values. Filtering is only available for public metadata.
+
+
+```graphql
```
Comment (krzysztofwolski) [inline docs/developer/metadata.mdx:149]:
```suggestion

```graphql
```

**Comment by patrys** [comment]
Task linked: [SALEOR-3216 Mention metadata filtering in the docs](https://app.clickup.com/t/2549495/SALEOR-3216)

File: docs/developer/metadata.mdx [hunk 2]
```diff
@@ -140,3 +140,43 @@ mutation {
   }
 }
 ```
+
+
+## Filter by metadata
```
Comment (karolkielecki) [inline docs/developer/metadata.mdx:145]: Filtering by metadata

Comment (karolkielecki) [inline docs/developer/metadata.mdx:145]:
```suggestion
## Filtering by metadata

File: docs/developer/metadata.mdx [hunk 3]
```diff
@@ -140,3 +140,43 @@ mutation {
...
+Objects with metadata interface can be filtered by it's values. Filtering is only available for public metadata.
```
Comment (karolkielecki) [inline docs/developer/metadata.mdx:146]:
```suggestion
Objects with metadata interface can be filtered by their values. Filtering is only available for public metadata.
```

**Review by karolkielecki — CHANGES_REQUESTED** [review]

**Review by korycins — APPROVED** [review]

**Review by karolkielecki — APPROVED** [review]

**Pull request merged**